R/interpret.R

```diff
@@ -117,7 +122,8 @@ Interpreter <- R6::R6Class(
     initialize = function(predictor = NULL,
                           samples = 1000,
                           data.points = NULL,
-                          grid.size = 50) {
+                          grid.size = 50,
+                          pdp.rank = 'Inflection Point') {
```

Comment:
A better name for this parameter would be `rank.method`, since we have several methods we will pass as an option. pdp.rank would be okay if it were just a boolean flag, but isn't descriptive enough in this case